hash.in

```diff
@@ -0,0 +1 @@
+1234
```

Comment:
I don't think you wanted to merge this, probably is a left over from a test you did.